mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java

```diff
@@ -114,14 +116,19 @@ public void onRefresh() {
     }
 
     private void setUpTriggersForced() {
+
+        if( getArguments().getString("state").compareTo("From Favourite") == 0)
```

Comment:
there should always be brackets around if and else. 

For consistency use opening bracket at same line of starting of block 

eg : `private void setUpFavTriggers() {`